Cargo.toml

```diff
@@ -35,6 +35,7 @@ wild = "2.0.*"
 globset = "0.4.*"
 xdg = "2.1.*"
 yaml-rust = "0.4.*"
+chrono = "0.4.19"
```

Comment:
Does `chrono` help all that much with speed up? Otherwise it would be better to not add an extra dependency.

Reply:
Regarding this, `chrono` is already a dependency due to `chrono_humanize` so this way, we get fir of the extra `time` dependency